jest.config.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
     '^.+\\.tsx?$': 'ts-jest'
   },
   // testRegex: 'spec\\.(ts|tsx)$' // TODO Skip api/ tests for now, as it's still WIP
-  testRegex: 'packages/(abi|light.js|light.js-react)/.*spec\\.(ts|tsx)$'
+  testRegex:
+    'packages/(abi|electron|light.js|light.js-react)/.*spec\\.(ts|tsx)$'
```

Comment:
still missing escaping the dot for `light.js` and `light.js-react` but it doesn't make a difference for all intents and purposes ^^